gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/resources/GenerateResourceAccessorsTask.kt

```diff
@@ -14,7 +14,18 @@ import java.io.File
 import java.nio.file.Path
 import kotlin.io.path.relativeTo
 
-internal abstract class GenerateResourceAccessorsTask : IdeaImportTask() {
+/**
+ * Defines a property responsible for enabling or disabling
+ * generation of @ResourceContentHash annotation for resource accessors.
+ */
+interface ResourceContentHashAnnotationGenerationConfiguration {
```

Comment:
Rename to `ResourceAccessorsConfiguration`

Reply:
done